src/components/app/MenuButtons.js

```diff
@@ -589,6 +604,69 @@ const MenuButtons = ({
   predictUrl,
 }: MenuButtonsProps) => (
   <div className="menuButtons">
+    <ButtonWithPanel
```

Comment:
I think it's worth breaking this out into its own component. All the other buttons inside `.menuButtons` already have their own component.